en/examples/offboard_velocity.md

```diff
@@ -152,16 +161,18 @@ inline void offboard_log(const std::string &offb_mode, const std::string msg)
  */
 bool offb_ctrl_ned(Device &device)
 {
+    std::shared_ptr<Offboard> offboard = std::make_shared<Offboard>(&device);
```

Comment:
This is actually wrong. We should not create `offboard` twice but only once and then pass it on as `std::shared_ptr<Offboard> offboard` instead of `Device &device`.

I guess we need to fix the example first.

Reply:
Update to example is here: https://github.com/dronecore/DroneCore/pull/246
Untested because I can't build dronecore https://github.com/dronecore/DroneCore/issues/245

I'll update docs shortly.